Sources/ClientRuntime/Config/SDKRuntimeConfiguration.swift

```diff
@@ -19,29 +19,3 @@ public protocol SDKRuntimeConfiguration {
     var retryer: SDKRetryer {get}
     var endpoint: String? {get set}
 }
-
-public extension SDKRuntimeConfiguration {
```

Comment:
This is a good example of when adding default implementations to protocols does not make sense. 
These defaults are all specific to the "default" or "crt based" configuration, and so we should be setting these values there and not at the protocol level.